airflow/www/static/js/graph.js

```diff
@@ -36,6 +36,7 @@ const dagRunId = getMetaValue('dag_run_id');
 const arrange = getMetaValue('arrange');
 const taskInstancesUrl = getMetaValue('task_instances_url');
 const isSchedulerRunning = getMetaValue('is_scheduler_running');
+const priority = getMetaValue('priority');
```

Comment:
Unfortunately this won't work.

To use `getMetaValue()`, we need to add a `<meta>` tag in `graph.html`, but that really only works for simple types like strings

Instead, I'd follow the example of `tasks`, `nodes`, `edges` in `graph/html` and then add `priority` as one of the global declaration at the top of this file.